src/app/query/bitcoin/transaction/transaction.query.ts

```diff
@@ -29,3 +33,26 @@ export function useGetBitcoinTransaction<T extends unknown = FetchBitcoinTransac
     ...options,
   });
 }
+
+const queryOptions = {
+  cacheTime: Infinity,
+  staleTime: 15 * 60 * 1000,
+  refetchOnWindowFocus: false,
+} as const;
+
+export function useGetBitcoinTransactionQueries(
```

Comment:
Seeing we are now primarily using naming of query hook files as `use-` rather than the `.query` syntax?

Reply:
Would like to know our overall pref here so I can rename if desired as I work in them. @kyranjamie

Reply:
Umm, I like `.query` but may myself have not followed this. Happy to stick to it, 👍🏼  for pushing for consistency